.github/workflows/release.yaml

```diff
@@ -0,0 +1,25 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release-type:
+        description: "The scope of the release (major, minor or patch)."
+        type: choice
+        required: true
+        default: patch
+        options:
+          - patch
+          - minor
+          - major
+
+jobs:
+  call-workflow-passing-data:
+    name: Java Gradle
+    uses: bakdata/ci-templates/.github/workflows/java-gradle-release.yaml@1.33.0
```

Comment:
Let's use the latest tag of the ci-templates in both workflows.

Reply:
No. This makes builds not reproducible in the future

Reply:
this PR is related to this [PR](https://github.com/bakdata/ci-templates/pull/141).
 am updating the `java-gradle-release` action and workflow right now. 

Updating `java-gradle-library` will be part of the following PR. because it will need to be coupled with a new PR in the `ci-templates` repo

Reply:
tag 1.33.0 does not exist 🤔
by latest I mean to set to the newest version: 
https://github.com/bakdata/ci-templates/tree/1.32.0

Reply:
It's a pre-bumped version then for the future release of the [Changelog PR](https://github.com/bakdata/ci-templates/pull/141)